openedx/features/edly/tests/test_middleware.py

```diff
@@ -61,10 +62,11 @@ def test_user_without_edly_organization_access(self):
         """
 
         with LogCapture(LOGGER_NAME) as logger:
-            response = self.client.get('/', follow=True)
-            assert response.status_code == 404
+            self.client.get('/', follow=True)
+            user = auth.get_user(self.client)
+            assert not user.is_authenticated()
 
-            logger.check(
+            logger.check_present(
```

Comment:
we also need to check if user is correctly redirected after logout.
We can use `redirect_chain`

https://docs.djangoproject.com/en/3.0/topics/testing/tools/
https://stackoverflow.com/questions/14951356/django-testing-if-the-page-has-redirected-to-the-desired-url

Reply:
Also update the docstring of this test case accordingly.

Reply:
Done